azext_iot/digitaltwins/_help.py

```diff
@@ -645,6 +655,16 @@ def load_digitaltwins_help():
             az dt twin relationship delete -n {instance_or_hostname} --twin-id {twin_id} --relationship-id {relationship_id} --etag {etag}
     """
 
+    helps["dt twin relationship delete-all"] = """
+        type: command
+        short-summary: Deletes all digital twin relationships on a Digital Twins instance, including incomming relationships.
+
+        examples:
+        - name: Delete all digital twin relationships associated with the twin.
+          text: >
+            az dt twin relationship delete-all -n {instance_or_hostname} --twin-id {twin_id}
```

Comment:
This is the only use case supported in the command right? As in you have to always provide a twin id. What about a scenario where I would like to clear all relationships but keep the twins in place? Also the command description sort of implies this is possible with the "Delete all digital twin relationships  on a Digital Twins instance," part. In a nutshell I am wondering if we could make the `--twin-id` optional.